src/services/auth.service.ts

```diff
@@ -0,0 +1,35 @@
+import axios from "axios/instanceAxios";
+
+const login = (email: string, password: string) => {
+  return axios
+    .post("/login", {
+      email,
+      password,
+    })
+    .then((res) => {
```

Comment:
async/await?

Reply:
przekażę Małgosi, bo to ona robiła. Stworzyłam po prostu swój nowy branch z brancha Gosi :)

Reply:
ok, w takim razie najlepiej poczekać, aż Gosia zmerguje pierwsza swoje zmiany

Reply:
ok, nie ma problemu :)